app/scripts/services/MaxMapMunicipalities.svc.js

```diff
@@ -0,0 +1,5 @@
+'use strict';
+
+angular.module('yourStlCourts').factory('MaxMapMunicipalities', function () {
```

Comment:
2 things. 

If this is only used in 1 spot, it doesn't need to be a separate angular "thing". Just define a var in your ctrl where it's needed. Also, it's not just "map" municipalities - it should apply to the dropdown even if they never go to the map itself. So maybe "MAX_SEARCH_MUNICIPALITIES" would be more appropriate.

If it DOES need to be used in multiple spots, instead of an angular factory, use an angular value or const. https://gist.github.com/demisx/9605099